File: Kotlin/PRIMERA RELACIÓN DE EJERCICIOS AFRICA/Primeros Ejercicios/src/ejercicio2.kt
/* Escribir un programa que almacene la cadena de caracteres contraseña en una
variable, pregunte al usuario por la contraseña e imprima por pantalla si la contraseña
introducida por el usuario coincide con la guardada en la variable sin tener en cuenta
mayúsculas y minúsculas. */
fun main() {
    val contrasenaG = "manteca1234"
    print("Introduce la contrasena: ")
    val contrasena = readLine()?.lowercase()

    if (contrasena != null && contrasena == contrasenaG) {
        println("La contrasena es correcta")
    } else {
        println("La contrasena es incorrecta")
    }
}

File: ProgPython/ActividadesT2/ejercicios2.1/condicionales/ejercicio2.py
def CompruebaContraseña (contraseña_introducida):
    if contraseña_introducida == contraseña_almacenada:
        return True
    else:
        return False
    

#PROGRAMA PRINCIPAL

contraseña_almacenada = "qwerty123"
contraseña_introducida = input("Introduce la contraseña: ")
contraseña_introducida = contraseña_introducida.lower()
if CompruebaContraseña (contraseña_introducida):
    print("La contraseña es correcta")
else:
    print("La contraseña es incorrecta")



